x-pack/plugins/observability/public/pages/slos/components/slo_sparkline.tsx

```diff
@@ -73,7 +59,19 @@ export function SloSparkline({ chart, data, id, isLoading, state }: Props) {
         theme={[theme, EUI_SPARKLINE_THEME_PARTIAL]}
       />
       <Tooltip type={TooltipType.None} />
-      {LineAxisComponent}
+      <Axis
+        id="axis"
+        hide
+        domain={{
+          min: NaN,
+          max: NaN,
+          fit: true,
+        }}
+        gridLine={{
+          visible: false,
+        }}
+      />
+      );
```

Comment:
`);` it looks like this is not needed there.

Reply:
Oops, forgot this thanks! Funny it does not show up on the page...

Reply:
Addressed in [02fde9d](https://github.com/elastic/kibana/pull/167521/commits/02fde9d9010f0183637ea3cbdbb8ef1d3a6e320e)